lib/core/notification/notification_service.dart

```diff
@@ -286,6 +288,15 @@ class NotificationService implements INotificationService {
     id ??= Random().nextInt(9999 - 1000) + 1000;
 
     await setupFlutterNotifications();
+
+    if (Platform.isAndroid) {
+      final androidInfo = await DeviceInfoPlugin().androidInfo;
```

Comment:
Do you think it would be good to surround this section with a try/catch in case getting the info fails?

Reply:
Can you recheck? :)

Reply:
I think in this way we can catch errors while fetching android info and errors while scheduling the notification. 

In all error cases i'm not scheduling the notification, just to be sure.. :)